components/switch/metadata/switch.yml

```diff
@@ -19,54 +20,104 @@ examples:
         <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-0">
         <span class="spectrum-Switch-switch"></span>
         <label class="spectrum-Switch-label" for="switch-onoff-0">Switch Off</label>
       </div>
-      <div class="spectrum-Switch">
+      <div class="spectrum-Switch spectrum-Switch--sizeM">
         <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-1" checked>
         <span class="spectrum-Switch-switch"></span>
         <label class="spectrum-Switch-label" for="switch-onoff-1">Switch On</label>
       </div>
 
       <br>
 
-      <div class="spectrum-Switch">
+      <div class="spectrum-Switch spectrum-Switch--sizeM">
         <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-disabled-0" disabled>
         <span class="spectrum-Switch-switch"></span>
         <label class="spectrum-Switch-label" for="switch-onoff-disabled-0">Switch Off</label>
       </div>
-      <div class="spectrum-Switch">
+      <div class="spectrum-Switch spectrum-Switch--sizeM">
         <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-disabled-1" checked disabled>
         <span class="spectrum-Switch-switch"></span>
         <label class="spectrum-Switch-label" for="switch-onoff-disabled-1">Switch On</label>
       </div>
+  - id: switch
+    name: Sizes
+    status: Verified
+    description: The t-shirt sizes of the Switch.
+    markup: |
+
+      <div class="spectrum-Switch spectrum-Switch--sizeS">
+        <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-0">
+        <span class="spectrum-Switch-switch"></span>
+        <label class="spectrum-Switch-label" for="switch-onoff-0">Switch Size S</label>
+      </div>
+
+      <br>
+
+      <div class="spectrum-Switch spectrum-Switch--sizeM">
+        <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-0">
+        <span class="spectrum-Switch-switch"></span>
+        <label class="spectrum-Switch-label" for="switch-onoff-0">Switch Size M</label>
+      </div>
+
+      <br>
+
+      <div class="spectrum-Switch spectrum-Switch--sizeL">
+        <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-0">
+        <span class="spectrum-Switch-switch"></span>
+        <label class="spectrum-Switch-label" for="switch-onoff-0">Switch Size L</label>
+      </div>
+
+      <br>
+
+      <div class="spectrum-Switch spectrum-Switch--sizeXL">
+        <input type="checkbox" class="spectrum-Switch-input" id="switch-onoff-0">
+        <span class="spectrum-Switch-switch"></span>
+        <label class="spectrum-Switch-label" for="switch-onoff-0">Switch Size XL</label>
+      </div>
   - id: switch
     name: Emphasis
     status: Verified
     description: The emphasized variant of the Switch.
     markup: |
-      <div class="spectrum-Switch spectrum-Switch--">
+      <div class="spectrum-Switch spectrum-Switch--sizeM spectrum-Switch--">
```

Comment:
```suggestion
      <div class="spectrum-Switch spectrum-Switch--sizeM spectrum-Switch--emphasized">
```

Reply:
oops, got to press that cmd + D better in VScode.